apps/condo/domains/resident/schema/ServiceConsumer.js

```diff
@@ -80,6 +80,25 @@ const ServiceConsumer = new GQLListSchema('ServiceConsumer', {
             kmigratorOptions: { null: true, on_delete: 'models.SET_NULL' },
         },
 
+        // todo(@toplenboren) DOMA-1701 Make this deprecated and add mobile an ability to move away from these fields
+        // The reason for this field is to avoid adding check for resident user into global AcquiringIntegrationContext read access.
+        // This field have specific use case for mobile client.
+        residentAcquiringIntegrationContext: {
+            schemaDoc: 'AcquiringIntegration, that is returned for current serviceConsumer in mobile client',
+            type: Virtual,
+            extendGraphQLTypes: ['type ResidentAcquiringIntegrationContext { id: ID!, integration: ID,  }'],
+            graphQLReturnType: 'ResidentAcquiringIntegrationContext',
+            resolver: async (item) => {
+                if (item.acquiringIntegrationContext) {
```

Comment:
```suggestion
                if (!item.acquiringIntegrationContext) return null
```
and then without else